aws_doc_sdk_examples_tools/config/services.yaml

```diff
@@ -2371,6 +2371,21 @@ outposts:
   tags:
     product_categories: {}
   version: outposts-2019-12-03
+partner-central:
+  long: '&AWS; &pcsh;'
```

Comment:
I don't see &pcsh; in either of the shared entity files. Are you sure this is correct?

Reply:
Done

Reply:
Turns out the validator doesn't like that, so I've moved the .ent file to this repo and added a local definition that we can use.